macros/vc.asm

```diff
@@ -0,0 +1,19 @@
+vc_hook: MACRO
+    if DEF(_CRYSTALVC)
+        .VC_\1::
+    endc
+ENDM
+
+
+vc_patch: MACRO
+    if DEF(_CRYSTALVC)
+        .VC_\1::
+    endc
+ENDM
+
+
+vc_assert: MACRO
+    if DEF(_CRYSTALVC)
+        assert \1, \2
```

Comment:
```suggestion
        assert \#
```

Reply:
If this macro is no longer used anywhere, just remove it until/unless we actually need it.